client/maisie/cli/display.py

```diff
@@ -0,0 +1,38 @@
+import click
+from terminaltables import SingleTable
+from maisie.utils.misc import Transform
+from colorama import Fore
+
+
+class Display:
```

Comment:
The `Display` class should be included in the `utils` module. `cli` is meant to contain only method groups and methods executed directly in the CLI, where `Display` is just a helper class used for managing the formatting of the response.